src/CMakeLists.txt

```diff
@@ -12,250 +12,9 @@ if ( "${isSysDir}" STREQUAL "-1" )
     set( CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib" )
 endif( "${isSysDir}" STREQUAL "-1" )
 
-# Find packages
-find_package(OpenGL     REQUIRED)
-find_package(Qt5 COMPONENTS Core Widgets OpenGL Xml REQUIRED)
+add_subdirectory( Core )
+add_subdirectory( Engine )
+add_subdirectory( IO )
+add_subdirectory( GuiBase )
 
-if(Qt5Core_VERSION VERSION_LESS 5.5 OR
-   Qt5Widgets_VERSION VERSION_LESS 5.5 OR
-   Qt5OpenGL_VERSION VERSION_LESS 5.5)
-    message(FATAL_ERROR "Qt5 or superior required (found ${Qt5Core_VERSION}).")
-else ()
-    message(STATUS "QT ${Qt5Core_VERSION} found." )
-endif()
-
-message(STATUS "Compiling Radium with 3rdparty include dirs : ${EIGEN3_INCLUDE_DIR} -- ${GLBINDING_INCLUDE_DIR} -- ${GLM_INCLUDE_DIR} -- ${GLOBJECTS_INCLUDE_DIR}-- ${OPENMESH_INCLUDE_DIR}" )
-
-if( RADIUM_ASSIMP_SUPPORT )
-    include_directories( ${ASSIMP_INCLUDE_DIR} )
-    message(STATUS "Compiling Radium with Assimp's include dir : ${ASSIMP_INCLUDE_DIR}" )
-endif( RADIUM_ASSIMP_SUPPORT )
```

Comment:
these 2 messages about 3rdParty directories have been removed

Reply:
Yes, this as been done in purpose.
The build chain will be updated in the next PR.